examples/Playground.hs

```diff
@@ -52,3 +66,15 @@ main = do
 
     putTextLn "\nFrom file configurations.."
     launchFromFile testLoggerConfigPath "node" runPlayLog
+
+#if ( __GLASGOW_HASKELL__ >= 802 )
+    launchSimpleLogging "concurrent" concurrentActions
+
+concurrentActions :: forall m . (WithLoggerIO m, MonadBaseControl IO m) => m ()
+concurrentActions = logWarningWaitInf 2 "Second" someStupidAction
```

Comment:
`"Second"` is the name of `someStupidAction`. So there will be something like
> Action `Second` took more than…

Wht `"Second"`? Why not `stupid action`, for example? It looks confusing.

Reply:
Oh, sorry there were two actions when I started to play with those functions and they were `First` and `Second`, and I forgot to change names to something that would make more sense.. Thanks for finding this out!